tox.ini

```diff
@@ -9,9 +9,6 @@ envlist =
     ; qt 5.9.7 (LTS)
     py36-{pyqt5,pyside2}_59_conda
```

Comment:
I am ok with testing PyQt5.9 in tox; but we should test it with python 3.7.  

I suggest removing all references to `py36` from here, and adding some `py39` references where appropriate.